org-formation/710-tgw/_tasks.yaml

```diff
@@ -114,6 +115,9 @@ TransitGatewayRoutes:
       # org-sagebase-synapsedw
       vpc:
         CIDR: "10.12.0.0/16"
+      # org-sagebase-sageit
+      sagedefaultvpc:
```

Comment:
i don't see a CFN stack in sageit called sagedefaultvpc, I do see a stack called defaultvpc. This should match the CFN stack name of the VPC, which in this case should probably be defaultvpc

Reply:
Good catch. Thx!